### PR TITLE
fix(core): drawing correctly the loader

### LINKED ISSUE #4773: 🐞 - Loader in Safari doesn't work as expected
### Playground Link

https://taiga-ui.dev/components/button

### Description

image

### Angular version

latest

### Taiga UI version

latest

### Which browsers have you used?

- [ ] Chrome
- [ ] Firefox
- [X] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #4773

## What is the new behavior?

image

### CODE REVIEW

**Comment by vladimirpotekhin** [comment]
Thickness changed in xs/xl sizes. Is this expected?

Снимок экрана 2023-08-28 в 14 19 53
Снимок экрана 2023-08-28 в 14 20 02

File: projects/core/components/loader/loader.style.less [hunk 1]
```diff
@@ -146,6 +146,13 @@
     overflow: hidden;
     transform: scale(1, -1);
     animation: tuiLoaderRotate 3s linear infinite;
+
+    /* Safari Only */
+    @supports (-webkit-hyphens: none) {
+        & {
+            overflow: visible;
```
Comment (waterplea) [inline projects/core/components/loader/loader.style.less:153]: Maybe it can be used in all browsers? What's the point of overflow: hidden in chrome?

Comment (splincode) [reply projects/core/components/loader/loader.style.less:153]: `What's the point of overflow: hidden`

if I use `overflow: visible` in chrome

image

But in Safari works as expected

**Review by vladimirpotekhin — APPROVED** [review]

**Comment by splincode** [comment]
@nsbarsukov hello, please review me

**Pull request merged**